ProcessMaker/Listeners/BpmnSubscriber.php

```diff
@@ -168,7 +169,8 @@ public function onIntermediateCatchEventActivated(IntermediateCatchEventInterfac
     {
         $messages = [
             MessageEventDefinitionInterface::class => 'System is waiting to receive message ":event"',
-            TimerEventDefinitionInterface::class => 'System has scheduled timer: ":event"'
+            TimerEventDefinitionInterface::class => 'System has scheduled timer: ":event"',
+            ConditionalEventDefinitionInterface::class => 'System is waiting for conditional event: ":event"',
```

Comment:
Correct the text to:
'System is waiting for _**the**_ conditional event: ":event"',